src/app/app-design-tool/components/app-form-design/app-form-design.component.html

```diff
@@ -0,0 +1,87 @@
+<!--Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.-->
+
+<!--Licensed under the Apache License, Version 2.0 (the "License");-->
+<!--you may not use this file except in compliance with the License.-->
+<!--You may obtain a copy of the License at-->
+
+<!--http://www.apache.org/licenses/LICENSE-2.0-->
+
+<!--Unless required by applicable law or agreed to in writing, software-->
+<!--distributed under the License is distributed on an "AS IS" BASIS,-->
+<!--WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.-->
+<!--See the License for the specific language governing permissions and-->
+<!--limitations under the License.-->
+
+<div class="container">
+  <form class="design-form" #designForm="ngForm">
```

Comment:
I think you should reconsider naming. Can you tell me why did you chose the names you eventually did?
If no, I suggest to call the module `desing-tool` instead of `app-design-tool` just like interactive map. After that you can go by `app-<component name>`. If the name would be to vague, you can add the module name `app-design-tool-form` but then follow the `app-<module>-<component>` naming convention.

Another consideration is to use only the module name within modules. The shorthand for design tool is `dt` so it would be `dt-form` I think I like this one the best, it's short, and it makes it ugly to reuse one component in another module, so you're forced to make it generic.